ssc_interface_wrapper_ros2/include/ssc_interface_wrapper/converter_node.hpp

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (C) 2022 LEIDOS.
```

Comment:
Since this is mostly a port of the ssc_interface it may be more appropriate to to 
```
/*
 * Copyright 2017-2019 Autoware Foundation. All rights reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
 
 /**
  * Modification (C) Leidos 2022
  * Updated ssc_interface to ROS2 as new converter_node
  */